05-integration-tests/readme.md

```diff
@@ -130,14 +129,14 @@ docker compose -f test-integration.yml up --force-recreate --exit-code-from buil
 
 If we run this, we will find that our service `build-db-relationships` is unable to find the postgres database:
 
-```
+```output
```

Comment:
Nice I didn't know about `output` on markdown.

Reply:
I added `output` because lint shows a warning when a code block (```) has no language.

![image](https://github.com/Lemoncode/github-actions-playground/assets/33926302/fd3b012e-588b-4bc1-8a1f-04f8e6475c08)

The list of supported languages/syntaxes:  https://github.com/jincheng9/markdown_supported_languages